config.toml

```diff
@@ -89,7 +89,7 @@ project_home = "https://openservicemesh.io/"
 
 # Version params
 osm_branch = "release-v1.2"
-osm_version = "v1.2.0-rc.1"
+osm_version = "v1.2.0"
```

Comment:
The latest version needs to be updated In L96

Reply:
@zr-msft has been making those changes in the rest of the release branches, but not sure I remember seeing one for the release-v1.2 branch. @zr-msft can you handle that if you don't already have a PR open?

Reply:
This hasn't been done on `main` either: https://github.com/openservicemesh/osm-docs/blob/main/config.toml
Could one of you update this ASAP so `docs.openservicemesh.io` correctly redirects to the release-v1.2 landing page and the dropdowns are accurate?

Thanks